src/main/kotlin/com/vk/admstorm/env/Env.kt

```diff
@@ -128,6 +130,15 @@ object Env {
             LOG.warn("Exception while deserialize data, further work of the plugin is impossible:", e)
             return
         }
+        val config = AdmStormSettingsState.getInstance()
+        if (config.localDeployConfig) {
+            val publishConfig = project.getService(PublishConfig::class.java)
+            val defServer = publishConfig.findDefaultServers().firstOrNull()
```

Comment:
I would look into `findDefaultServerOrGroup`, `findDefaultServers` might return something if there is no default server